project/src/main/webapp/script.js

```diff
@@ -210,12 +213,14 @@ function getSecurityQuestions(){
 function getQuestion(question) {    
     const questionElement = document.createElement("li");
     questionElement.style.listStyle = "none";
+    questionElement.className = "li";
```

Comment:
Can you just set the style on li elements rather than setting a classname? Or do you not want the styles applied to all li elements? (In which case, it would be better to name the class something more descriptive.)